lib/debug.nix

```diff
@@ -206,86 +206,63 @@ rec {
 
   # -- DEPRECATED --
 
-  traceShowVal = x: trace (showVal x) x;
-  traceShowValMarked = str: x: trace (str + showVal x) x;
+  traceShowVal =
+    throw ( "`traceShowVal` is deprecated "
+          + "and will be removed in release 23.05. "
+          + "Please use `FILL IN`." );
```

Comment:
Again, if there was no warning already, it should become a warning, not an error.

Reply:
These used showVal which had a warning already so imo its good enough.

Reply:
Ahhh! Sorry for that then, makes sense